consensus/istanbul/core/handler_test.go

```diff
@@ -464,8 +518,331 @@ func TestCore_handlerMsg(t *testing.T) {
 	}
 }
 
+// TODO-Klaytn: To enable logging in the test code, we can use the following function.
+// This function will be moved to somewhere utility functions are located.
+func enableLog() {
```

Comment:
@ehnuje I was thinking the same. Perhaps in `tests/testutil`?
But since enableLog() is already in two places (snapshot/generate_test.go and tests/smartcontract_creation_test.go) we can refactor enableLog() in a separate PR.

Reply:
@ehnuje Please refer to https://github.com/klaytn/klaytn/pull/1402.